lib/Doctrine/Migrations/Tools/Console/Command/CurrentCommand.php

```diff
@@ -15,7 +15,7 @@
  */
 final class CurrentCommand extends DoctrineCommand
 {
-    /** @var string */
+    /** @var string|null */
```

Comment:
Why are these nullable?

Reply:
Because they are in the parent class and types of overridden properties have to be invariant.